example/helm/values.kubeflow.yaml

```diff
@@ -19,6 +19,21 @@ pipelines:
         secretKeyRef:
           name: mlpipeline-minio-artifact
 
+katib:
+  dbmanager:
+    config:
```

Comment:
Does it make sense to create default db configuration since we use the same secrets/host for
1) pipeline component
2) katib
And add ability to override for each component ? I see this is quite common approach to avoid repeating the same configs

Reply:
This is a good idea but let's handle that as a separate issue.

https://github.com/kromanow94/kubeflow-manifests/issues/9